.github/workflows/ci.yml

```diff
@@ -26,6 +26,9 @@ jobs:
         include:
           - os: ubuntu-latest
             run_supabase_tests: true
+          - os: ubuntu-latest
+            run_supabase_tests: false
+            upload-service: true
```

Comment:
Remind me (again): Why is this not set to false at the top-level matrix like `run_supabase_tests`?

Reply:
Because we would then add a new combination and have two 'ubuntu-latest/false'. One with upload-service set, and one without.